lib/consumer/amqp_data_consumer/connection.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Mississippi.Consumer.AMQPDataConsumer.Connection do
```

Comment:
Nit: at the moment only AMQP is supported, so I'd call this
```suggestion
defmodule Mississippi.Consumer.AMQPDataConsumer.AMQPConnection do
```